ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/snippet/factory/BasicSnippetFactory.java

```diff
@@ -136,7 +136,7 @@ public VariableDeclarationSnippet createVariableDeclarationSnippet(Node node) {
                 qualifiers = NodeFactory.createNodeList(varNode.finalKeyword().get());
             }
             dclnNode = NodeFactory.createModuleVariableDeclarationNode(
-                    NodeFactory.createMetadataNode(null, varNode.annotations()),
+                    NodeFactory.createMetadataNode(null, varNode.annotations()), null,
```

Comment:
We probably need to introduce changes to the shell for this change, right? Can you please check with @kdsuneraavinash and create an issue if needed?

Reply:
I think there is a separate issue for the public variable support for the shell. Will check